electron-app/src/store/balances/getters.ts

```diff
@@ -112,6 +114,76 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, Zero);
   },
 
+  // simplify the manual balances object so that we can easily reduce it
+  manualBalanceByLocation: (
+    state: BalanceState,
+    balanceGetters,
```

Comment:
if you only use `exchangeRate` and `fiatTotal` you could do a { exchangeRate, fiatTotal } here instaed of passing the whole object